CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ This change log adheres to standards from [Keep a CHANGELOG](https://keepachange
 - add [`enforce-node-protocol-usage`] rule and `import/node-version` setting ([#3024], thanks [@GoldStrikeArch] and [@sevenc-nanashi])
 - add TypeScript types ([#3097], thanks [@G-Rath])
 
+### Fixed
+- [`no-unused-modules`]: don't error out when running with flat config and an eslintrc isn't present ([#3116], thanks [@michaelfaith])
```

Comment:
I think this changelog entry will need to be revised, but I didn't want to adjust that since you added it.

Reply:
totally fine either way; the format should hopefully be self-explanatory based on the commit message

Reply:
Updated